lib/manager/helm-values/util.ts

```diff
@@ -30,3 +30,10 @@ export function matchesHelmValuesDockerHeuristic(
     hasKey('tag', data)
   );
 }
+
+export function matchesHelmValuesInlineImage(
+  parentKey: string,
+  data: unknown
+): boolean {
```

Comment:
```suggestion
): data is string {
```
So this function can be used as type check